### ISSUE TITLE
validators weblets usage of threefold binance node

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/grid_weblets#875]
while specifying the ethereum rpc URL we are using the bsc seed node which is subject to rate limits, please allow the user to enter the field or if empty to use one of the threefold bsc public nodes, and clarify that in the description

@hossnys please provide the url for the threefold bsc full node

**Comment by Mahmoud-Emad** [comment]
@xmonader @AhmedHanafy725 
So we can set this node URL as a default value, when the user wants to change it he can change it, other ways we take the default value.
I suggest this option to make a condition on the input field to take only the URL
agree with me??

**Comment by xmonader** [comment]
I suggest we have the ethereum RPC text field to have this message `your full Ethereum node or we will use one of Threefold` and if the user leaves it empty we use one of the threefold's full nodes.

**Comment by coesensbert** [comment]
Current addresses of our BSC full nodes: https://docs.grid.tf/threefold/itenv_threefold_main/src/branch/master/Grid-deployments/Mainnet/bsc-full-node.md#current-nodes

**Comment by Mahmoud-Emad** [comment]
@coesensbert 

it gave me 404, permission is required or ??

![image](https://user-images.githubusercontent.com/57001890/177038023-3d258e1d-6281-4b21-8c5e-e76f609f8bbe.png)

**Comment by coesensbert** [comment]
> 

Yes, create your account at https://docs.grid.tf/ by logging in with the tf-connect app.
I'll give you permissions afterwards.

**Comment by coesensbert** [comment]
> @coesensbert Yes I already have one its username is `emad`

Permission added
